docs/reference-guides/core-blocks.md

```diff
@@ -420,7 +420,7 @@ A collection of blocks that allow visitors to get around your site. ([Source](ht
 
 -	**Name:** core/navigation
 -	**Category:** theme
--	**Supports:** align (full, wide), ariaLabel, inserter, interactivity, layout (allowSizingOnChildren, default, ~~allowInheriting~~, ~~allowSwitching~~, ~~allowVerticalAlignment~~), spacing (blockGap, units), typography (fontSize, lineHeight), ~~html~~
+-	**Supports:** align (full, wide), ariaLabel, inserter, interactivity, layout (allowSizingOnChildren, default, ~~allowInheriting~~, ~~allowSwitching~~, ~~allowVerticalAlignment~~), spacing (blockGap, units), typography (fontSize, lineHeight), ~~blockRenaming~~, ~~html~~
```

Comment:
I do not understand the changes to these files. They were created when I ran `npm run docs:build` but they seem totally wrong. 

Why would some blocks be marked as included `blockRenaming` support and not others.

Moreover, some blocks such as Pattern which do not support blockRenaming (i.e. it's set to `false` in its `block.json`) show up as having support.

Reply:
I think the `~~blockRenaming~~` syntax means that the block doesn't support that thing (notice that it's the same for `~~html~~` which is also set to false in the block.